### PR TITLE
Issue #129 Add scenario for removal of Che addon

### PR DESCRIPTION
**Pull request by agajdosi**
Fix https://github.com/minishift/minishift-addons/issues/129.

### CODE REVIEW

File: test/integration/features/che.feature [hunk 1]
```diff
@@ -1,53 +1,53 @@
-@che
+@che @addon-che @addon
```
Comment (amitkrout) [inline test/integration/features/che.feature]: Any specific reason to keep both tag ```@addon-che``` and  ```@addon```

File: test/integration/features/che.feature [hunk 2]
```diff
@@ -59,6 +59,11 @@ Feature: Che add-on
     | PHP                   | https://github.com/che-samples/web-php-simple.git                        |
     | C++                   | https://github.com/che-samples/console-cpp-simple.git                    |
 
+  Scenario: Removal of Che addon
```
Comment (amitkrout) [inline test/integration/features/che.feature]: +1

File: test/integration/features/che.feature (hunk 1 above)
Comment (agajdosi) [reply test/integration/features/che.feature]: for later integration where we also use addon-eap-cd, not only eap-cd

Comment (amitkrout) [reply test/integration/features/che.feature]: ahh..sorry i suppose to say ```@che``` and ```@addon-che```

Comment (LalatenduMohanty) [reply test/integration/features/che.feature]: Yeah `@che` and `@addon-che` seems redundant.

**Comment by LalatenduMohanty** [comment]
@agajdosi the tests are failing. https://ci.centos.org/job/minishift-addons-pr/46/console

**Comment by agajdosi** [comment]
@LalatenduMohanty thanks for noticing. Seems like there were some commits to integration tests by Praveen, which broke the functionality with 1.21.0 release, so I have updated the constraint to use versions instead of latest master. Meaning we will use latest released binary here and test code from the same release. Will see how it works on the centos-ci.

**Comment by agajdosi** [comment]
@LalatenduMohanty issue fixed. Passed on centos-ci + all addons pass locally. I think the issue with the way how we vendor main minishift testsuite package does not need a new issue, so I am keeping it in this PR, lets take a look how you see it :)

File: Gopkg.toml [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 [[constraint]]
   name = "github.com/minishift/minishift"
-  branch = "master"
+  version = ">=1.21.0"
```
Comment (LalatenduMohanty) [inline Gopkg.toml]: @agajdosi This is not a real fix IMO. Because when we move to minishift version 1.22.0 these tests will gain fail right?

Comment (LalatenduMohanty) [reply Gopkg.toml]: Or does it need to be same with the version of minishift binary is run for the tests.

Comment (budhram) [reply Gopkg.toml]: @LalatenduMohanty Probably `>=` will take care. Isn't it @agajdosi ?

Comment (agajdosi) [reply Gopkg.toml]: AFAIU yes, checked the `dep` docs once more and it seems like this will take each following release.

Comment (praveenkumar) [reply Gopkg.toml]: @agajdosi why don't we use master branch itself so that nightly test for add-on also run as expected?

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
@praveenkumar @LalatenduMohanty Changes implemented.

**Comment by agajdosi** [comment]
@minishift/minishift-dev please take a look when you have time

**Comment by agajdosi** [comment]
@minishift/minishift-dev please take a look when you have time

**Comment by agajdosi** [comment]
@minishift/minishift-dev can somebody merge, please?

**Comment by agajdosi** [comment]
Closing this as there is no will to merge.

File: test/integration/features/che.feature [hunk 4]
```diff
@@ -59,6 +59,11 @@ Feature: Che add-on
...
+     When executing "minishift addons remove che" succeeds
+      And executing "oc get projects" succeeds
+     Then stdout should not contain "mini-che"
```
Comment (gbraad) [inline test/integration/features/che.feature]: 👍

**Comment by gbraad** [comment]
@LalatenduMohanty did you look at this after the requested change?

File: Gopkg.lock [hunk 5]
```diff
@@ -63,8 +57,8 @@
     "libmachine/state",
     "version"
   ]
-  revision = "9ba6da9ebd1140b65eb73e1ce08086ca72764c60"
-  version = "v0.13.0"
+  revision = "b48dc28d9139c93d166f07d8b3a049b59bceef9c"
+  version = "v0.15.0"
```
Comment (praveenkumar) [inline Gopkg.lock]: :+1:

**Pull request merged**